client/app/stores/router_store.coffee

```diff
@@ -186,7 +186,12 @@ class RouterStore extends Store
             @emit 'change'
 
         handle ActionTypes.ROUTE_CHANGE, (value) ->
-            _action = value
+            # We cant display any informations
+            # without accounts
+            unless AccountStore.getAll()?.size
+                _action = AccountActions.CREATE
```

Comment:
this patch was done according to this only one `action` dispatched. The tricky case is when user has no account, and the url is `http://localhost:9125/` where `value.action` is equal to `AccountActions.SHOW_ALL`. In this case no messages could be found. Changing the param here make the redirection : I didn't fount a better place to do this.